contracts/parent-chain/contract-upgrades/example-env/.env-espresso-osp-migration.example

```diff
@@ -0,0 +1,6 @@
+export NEW_OSP_ENTRY="0x7B5F0B437EE68A22992DdD629AA22525Fc5dfa9A"
```

Comment:
Nit: I dont think you need `export` in front of the environment variables ( example [here](https://github.com/EspressoSystems/nitro-contracts/blob/integration/.env.sample.goerli#L1))

Reply:
If the file is called .env exactly it is special and automatically picked up by foundry and many other tools. Unfortunately it has to be exactly .env in the case of foundry.

Reply:
you can do `source .env-espresso-osp-migration && forge test <>` to be able to have a foundry script pick up env variables that's not in the expected `.env` file.

Reply:
That's true but then breaks the ability of being able to override with env vars. So this can get messy when used in scripts if one expects environment variable that are already set to take precedence.

Reply:
that's true. we aren't using the .env file for secrets anymore but only for config (since that's how it's been set up historically). So should this go into the .env file?